client/components/animations.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Canvas from './canvas';
+
+export default function Animations(props) {
+  return (
+    <>
+      <Canvas />
+    </>
+  );
+}
```

Comment:
If this Animation component won't have a significant role in the sprite canvases, it's probably better to remove it entirely and stick with the Canvas component only